examples/tests/valid/container_types.w

```diff
@@ -18,4 +18,10 @@ let m2: Map<num> = {"a":1, "b":2, "c":3};
 let m3 = Map<num> {"a":1, "b":2, "c":3};
 let m4: Map<num> = Map<num> {"a":1, "b":2, "c":3};
 let m5 = {"a":bucket1, "b":bucket2, "c":bucket3};
-let m6: Map<cloud.Bucket> = {"a":bucket1, "b":bucket2, "c":bucket3};
\ No newline at end of file
+let m6: Map<cloud.Bucket> = {"a":bucket1, "b":bucket2, "c":bucket3};
+
+//Set tests
+let s2: Set<num> = {1, 2, 3};
+let s3 = Set<num> {1, 2, 3};
+let s4: Set<num> = Set<num> {1, 2, 3};
+let s6: Set<cloud.Bucket> = {bucket1, bucket2, bucket3};
```

Comment:
Curious- i am assuming JS used referential equality to determine uniqueness. Correct? Does that mean that `add()` is o(n)? (We have to check if each element is already there)

Reply:
I don't think it checks if an element is already there. It just overrides it

Reply:
But there isn't really a formal concept of a hashcode in JavaScript.

The [spec] is very illusive.

[spec]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set#performance